src/material/core/focus-indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,95 @@
+@import '../theming/theming';
+
+/// Mixin that turns on strong focus indicators.
+///
+/// @example
+///   .my-app {
+///     @include mat-strong-focus-indicators();
+///   }
+
+@mixin mat-strong-focus-indicators() {
+  // Reference to the root selector, used to simplify the nested selectors
+  // in this mixin.
+  $root-selector: #{&};
```

Comment:
I mentioned this in Slack, but I came back to this idea and inverted it to repeat `.mat-focus-indicator` instead of the `$root-selector`, which I think makes it a little less strange overall: https://www.sassmeister.com/gist/1a591faf77e1f984090e57c2efffe964

Could do this here or in a follow-up

Reply:
Sounds good, I'll just update it here.